release/lts/debian9/docker/Dockerfile

```diff
@@ -41,7 +41,6 @@ RUN apt-get update \
     && locale-gen && update-locale \
     # remove powershell package
     && rm /tmp/powershell.deb \
-    && ln -sf /opt/microsoft/powershell/7-preview/pwsh /usr/bin/pwsh \
```

Comment:
unlike other Debians in this PR, here creation of '/usr/bin/pwsh' sym link is remoevd and `/usr/bin/pwsh-lts` is not created. Why?

Reply:
same for all lts ubuntu dockerfiles in this PR.

Reply:
The deb file creates it here.